apps/src/applab/designMode.js

```diff
@@ -840,7 +840,7 @@ function makeDraggable(jqueryElements) {
         });
 
         // Dim the element if it's dragged out of bounds
-        if (!isMouseInBounds(newLeft, newTop)) {
+        if (!isMouseInBounds(newLeft + event.offsetX, newTop + event.offsetY)) {
```

Comment:
I stopped to read the implementation of `isMouseInBounds` because it seemed odd to handle the cursor offset here, outside of such a function.  Turns out we should probably rename that helper something like `isPointInBounds` because it has no actual knowledge of the mouse, as such.  Thoughts on doing that in this PR?